src/editor/actions/history-modal-actions.js

```diff
@@ -48,6 +49,22 @@ export function updateSelectedRevisionId(selectedRevisionId) {
   };
 }
 
+export function revertToSelectedRevisionId() {
+  return (dispatch, getState) => {
+    dispatch({
+      type: "UPDATE_IOMD_CONTENT",
+      iomd: getState().notebookHistory.revisionContent[
+        getState().notebookHistory.selectedRevisionId
+      ]
+    });
```

Comment:
It occurs to me also that this won't restore the title of the previous revision, but again, it might be better to wait for that until we show the title in this dialog.